src/geo/projection/globe_util.js

```diff
@@ -181,7 +181,13 @@ export function aabbForTileOnGlobe(tr: Transform, numTiles: number, tileId: Cano
     }
 
     if (bounds.contains(tr.center)) {
+        // Extend the aabb by encapsulating the center point
         cornerMax[2] = 0.0;
```

Comment:
Is this line still needed?

Reply:
Yes, this is needed for example for large tiles whose corner points are behind the camera far plane